src/site/antora/modules/ROOT/pages/artifacts.adoc

```diff
@@ -0,0 +1,585 @@
+////
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+////
+
+= Artifact reference
```

Comment:
As I stated earlier, I am a little bit reluctant with the word `Artifact`, in particular, in the menu. We always referred to our support _artifacts_/_modules_ as _components_. I don't see a compelling reason to move away from that nomenclature.

```suggestion
= Components
```